### PR TITLE
SignalR fixes

### PR DESCRIPTION
**Pull request by adrnchk**

### CODE REVIEW

File: DogeNet.WebApi/Startup.cs [hunk 1]
```diff
@@ -103,6 +105,7 @@ public void ConfigureServices(IServiceCollection services)
                     );
             });
             services.AddSignalR();
+            services.AddSingleton<IDictionary<string, UserConnection>>(opts => new Dictionary<string, UserConnection>());
```
Comment (oleh-leobit) [inline DogeNet.WebApi/Startup.cs:108]: What is this for?

File: DogeNetWebClient/src/app/app.component.ts [hunk 2]
```diff
@@ -17,10 +17,8 @@ export class AppComponent implements OnInit {
     this.oidcSecurityService
       .checkAuth()
       .subscribe(({ isAuthenticated, userData }) => {});
-
     this.oidcSecurityService.getAccessToken().subscribe((token) => {
-      this.signalrService.startConnection(token);
-      this.signalrService.addMessageListener();
+      this.signalrService.joinRoom(token, 'us1', 'g1');
```
Comment (oleh-leobit) [inline DogeNetWebClient/src/app/app.component.ts:21]: Const string?

File: DogeNetWebClient/src/app/messenger/pages/chat-box/chat-box.component.ts [hunk 3]
```diff
@@ -51,13 +51,20 @@ export class ChatBoxComponent implements OnInit {
     this.conversationsService
       .apiConversationGetConversationByIdIdGet$Json({ id: this.id })
       .subscribe((res) => (this.conversationInfo = res));
+    // this.signalrService.hubConnection?.on(
```
Comment (oleh-leobit) [inline DogeNetWebClient/src/app/messenger/pages/chat-box/chat-box.component.ts]: Comments

**Review by oleh-leobit — CHANGES_REQUESTED** [review]

File: DogeNetWebClient/src/app/messenger/pages/chat-box/chat-box.component.ts (hunk 3 above)
Comment (adrnchk) [reply DogeNetWebClient/src/app/messenger/pages/chat-box/chat-box.component.ts]: deleted

File: DogeNetWebClient/src/app/app.component.ts (hunk 2 above)
Comment (adrnchk) [reply DogeNetWebClient/src/app/app.component.ts:21]: @oleh-leobit this data should be stored in ngrx storage, I will change it as soon as posible

File: DogeNet.WebApi/Startup.cs (hunk 1 above)
Comment (adrnchk) [reply DogeNet.WebApi/Startup.cs:108]: @oleh-leobit this is signalR connections dictionary, is used here
image

**Pull request merged**